devdocs-generator.ps1

```diff
@@ -0,0 +1,195 @@
+<#
+
+    .DESCRIPTION
+    This script generates markdown files for the development documentation based on the existing JSON files.
+    Create table of content.
+
+#>
+
+# Load the JSON files
+$tweaks = Get-Content -Path "config/tweaks.json" | ConvertFrom-Json
+$features = Get-Content -Path "config/feature.json" | ConvertFrom-Json
+
+# Create the output directories if they don't exist
+$tweaksOutputDir = "docs/dev/tweaks"
+$featuresOutputDir = "docs/dev/features"
+
+if (-Not (Test-Path -Path $tweaksOutputDir)) {
+    New-Item -ItemType Directory -Path $tweaksOutputDir | Out-Null
+}
+
+if (-Not (Test-Path -Path $featuresOutputDir)) {
+    New-Item -ItemType Directory -Path $featuresOutputDir | Out-Null
+}
+
+# Function to generate markdown files
+function Generate-MarkdownFiles($data, $outputDir, $jsonFilePath, $type) {
+    $tocEntries = @()
+
+    foreach ($itemName in $data.PSObject.Properties.Name) {
+        $itemDetails = $data.$itemName
```

Comment:
Adding a check at the beginning for things that don't make sense to "document", like some Json Entry "glue", a good example are json entries with the `Button` Type assigned to them, _see screenshot below to see why I'm giving this suggestion in first place_.

![image](https://github.com/user-attachments/assets/038a72d0-26c4-4ee5-8883-16c6661e6b8e)

Reply:
... The entry doesn't give that much detail, and if we _really_ want to document everything.. then some types of entries should be separate to be something else, as they're there to make the winutil code function properly.

Reply:
about entrys like the shortcut, for those I made places to add custom content that won't be overwritten, these are made to explain difficult stuff manually better or custom stuff that only envolves scripts, or even nothing like the shortcut example you gave.

Reply:
can I mark this convo as resolved or do you want to remove these items entirely?

Reply:
I think there's room for improvement on this end, because the design in general doesn't make sense (the json parsing & making UI from it, not your PR).. so we could revisit this part in the near future, for now.. it's good for now, meanwhile we could add custom content to better explain these entries, which's quite a nice feature I must say, thanks Drift 👍